lesson35/lesson35_part2.py

```diff
@@ -14,7 +14,6 @@ def car_info(self):
         method returns car info
         :return: string of car info
         """
-
         return (f'{self.model} {self.year}:'
                 f'\nEngine - {self.engine}'
                 f'\nMileage - {self.mileage} км'
```

Comment:
def compare_mileages(self, other_car):
For this method also better return some value True/False or Car with largest milages, but not string